packages/@n8n/nodes-langchain/nodes/Guardrails/actions/checks/topicalAlignment.ts

```diff
@@ -0,0 +1,12 @@
+import { createLLMCheckFn } from '../../helpers/model';
+import type { CreateCheckFn, LLMConfig } from '../types';
+
+export const TOPICAL_ALIGNMENT_SYSTEM_PROMPT = `You are a content analysis system that determines if text stays on topic.
+
+BUSINESS SCOPE: [INSERT BUSINESS SCOPE HERE]
```

Comment:
I wonder if it would be nicer UX if the business scope was a separate required text input, and this check had a "Customize Prompt" toggle like some of the other checks.